static/js/repo_detail.js

```diff
@@ -48,3 +48,51 @@ $('.markdown-body a:not(.anchor)').each(function(idx, value) {
     value.target = '_blank';
   }
 });
+
+function getParameterByName(name, url) {
```

Comment:
I would also suggest moving this into a different file, since it's more general and include it here.